### PR TITLE
Mail articles by last 5 users

### PR DESCRIPTION
**Pull request by mrosso10**
# Summary
* Added functionality to send email with last 10 articles from last 5 users

# Test plan
1. Start the sidekiq process with **bundle exec sidekiq**
2. Open the home page
3. Click on the "Send last articles from last 5 users by email" link
4. Check your email

# Trello card
https://trello.com/c/ZFZyEKuR/15-crear-un-email-con-los-ultimos-10-articulos-creados-por-las-5-ultimas-personas-en-registrarse

### CODE REVIEW

File: app/mailers/blog_mailer.rb [hunk 1]
```diff
@@ -5,4 +5,10 @@ def last_articles(to)
          to: to,
          from: 'blog@wolox.com.ar'
   end
+  def articles_from_last_users(to)
+    @users = User.last 5
+    mail subject: 'Check the 10 articles from last 5 users',
```
Comment (Dinuuu) [inline app/mailers/blog_mailer.rb]: intenationalize the subject of the mail

File: app/views/layouts/_navbar.html.slim [hunk 2]
```diff
@@ -12,6 +12,8 @@
         ul.nav.navbar-nav
           = form_tag send_articles_email_path do
             input type="submit" value="Send last 10 articles by email"
```
Comment (Dinuuu) [inline app/views/layouts/_navbar.html.slim]: You should internationalize the buttoms in the views

File: config/routes.rb [hunk 3]
```diff
@@ -2,7 +2,9 @@
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
   devise_for :users
+
   post "articles/send_articles_email", as: "send_articles_email"
+  post "articles/articles_from_last_users", as: "articles_from_last_users"
```
Comment (Dinuuu) [inline config/routes.rb]: same correction as in the other PR ( about the name of the endpoint and that can be put inside the `resources :articles do `

File: config/routes.rb [hunk 4]
```diff
@@ -2,10 +2,14 @@
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
   devise_for :users
+
   post "articles/send_articles_email", as: "send_articles_email"
 
   resources :articles do
     resources :comments
+    collection do
+      post :articles_from_last_users
```
Comment (Dinuuu) [inline config/routes.rb]: This route should be a get since you are just getting info of the database and not modifying anything

**Comment by Dinuuu** [comment]
Test a little this new resource (You can test a new job is enqueded )

**Comment by Dinuuu** [comment]
Check that the build is not passing

**Comment by mrosso10** [comment]
That's because the Travis configuration is on another branch. What should I do?

**Comment by mrosso10** [comment]
Now passing tests

**Comment by Dinuuu** [comment]
Squash and :+1:

**Pull request merged**